### PR TITLE
Chore - Codeowners Updated

### PR DESCRIPTION
**Pull request by seba-aln**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,2 @@
-*         @seba-aln @MikeDobrzan @marcin-cebo @mohitpubnub
-.github/* @parfeon @seba-aln @MikeDobrzan @marcin-cebo @mohitpubnub
```
Comment (marcin-cebo) [inline .github/CODEOWNERS:2]: Doen't Serhii require access to .github folder?

Comment (parfeon) [reply .github/CODEOWNERS:2]: `*` covers all folders. If I change something, then will need someone from the first line to approve it ;)

Comment (seba-aln) [reply .github/CODEOWNERS:2]: don't worry it was agreed with Serhii :) You really think I could strip any privileges from him?

**Review by parfeon — APPROVED** [review]
LGTM!

**Pull request merged**